infra/terraform/modules/auth0/rules/Multifactor-Google-Authenticator-Do-Not-Rename.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * This rule is managed by Terraform, DO NOT EDIT!
+ */
+function (user, context, callback) {
+    var rangeCheck = require('range_check');
+
+    var AUTHENTICATOR_LABEL = 'MOJ Analytical Platform (Dev)';
+    var CONNECTION = user.identities[0].connection;
+    var ENABLED_CONNECTIONS = [
```

Comment:
This should be a variable, with its contents taken from the declared client resources

```
var ENABLED_CONNECTIONS = [
  "${auth0_connection.github.name}"
]
```

Or similar - you'd actually need to use a `var.foo` here if this is changed to a template file, with the list of connections passed in to the `template_file` resource.